resources/function_help/json/to_string

```diff
@@ -5,11 +5,11 @@
   "description": "Converts a number to string.",
   "arguments": [{
     "arg": "number",
-    "description": "Integer or real value. The number to convert to string."
+    "description": "Integer or real value. The number to convert to string. The conversion is not locale-aware, see 'format_number' for a locale-aware alternative."
```

Comment:
Wouldn't it be better to add the note to the description, instead?

Reply:
done https://github.com/qgis/QGIS/commit/7c5db3b3077b85a2234d84142772dbc80104acad